src/Faker/Provider/uk_UA/Address.php

```diff
@@ -130,7 +130,7 @@ class Address extends \Faker\Provider\Address
         'Чорногорія',
         'Марокко',
         'Мозамбік',
-        'М\'янма',
+        'М’янма',
```

Comment:
Do you mean that ' (the ascii quote) isn't at all used in Ukrainian, or is it a matter of typography?